src/runtime/pkg/device/drivers/utils.go

```diff
@@ -45,9 +45,9 @@ func deviceLogger() *logrus.Entry {
 	return api.DeviceLogger()
 }
 
-// Identify PCIe device by reading the size of the PCI config space
+// IsPCIeDevice Identify PCIe device by reading the size of the PCI config space
 // Plain PCI device have 256 bytes of config space where PCIe devices have 4K
-func isPCIeDevice(bdf string) bool {
+func IsPCIeDevice(bdf string) bool {
 	if len(strings.Split(bdf, ":")) == 2 {
```

Comment:
By capitalizing the function `IsPCIeDevice` name, you [make it exported](https://yourbasic.org/golang/public-private/) (yeah, I know, one of these go-isms I really don't like), but I see nothing in the commit requiring this. Do you plan to use that function from another module later? Otherwise, let's not make it part of the public interface.

Reply:
IsPCIeDevice is used in different modules so yes I need it.